packages/proxy/package.json

```diff
@@ -36,6 +36,7 @@
     "@packages/server": "0.0.0-development",
     "@types/express": "4.17.2",
     "@types/supertest": "2.0.10",
+    "devtools-protocol": "0.0.927104",
```

Comment:
This may need to be nohoisted or it could lead to type errors liked I found with https://github.com/cypress-io/cypress/pull/28372

Reply:
Good catch: [`bf92f82` (#28517)](https://github.com/cypress-io/cypress/pull/28517/commits/bf92f825c87e6502deb70ce56631877b39c758b4)